docs/MIGRATION_GUIDE_08.md

```diff
@@ -0,0 +1,238 @@
+# Migration guide matter.js Device API to 0.8
+
+With version 0.8 the matter.js project introduces a new high level API to build devices bases on code generation
+for all Matter 1.1 device types, clusters and functionality. In the past we already generated the code for all 
+cluster definitions and this got now enhanced for all device types. Additionally, the way how own cluster logic can be 
```

Comment:
cluster definitions and this has now been enhanced for all device types. Additionally, the way custom cluster logic can be